src/main/scala/mesosphere/marathon/api/v2/validation/SchedulingValidation.scala

```diff
@@ -164,6 +170,7 @@ object SchedulingValidationMessages {
   val InvalidRegularExpression = "is not a valid regular expression"
   val ConstraintLikeAnUnlikeRequireRegexp = "LIKE and UNLIKE require a non-empty, regular expression value"
   val ConstraintMaxPerRequiresInt = "MAX_PER requires an integer value"
+  val ConstraintMaxPerRelativeRequiresDouble = "MAX_PER_RELATIVE requires a double value"
```

Comment:
I think we should specify the range of this value since only values between 0 and 1 are relevant